src/model_service.cpp

```diff
@@ -47,10 +47,21 @@ void addStatusToResponse(tensorflow::serving::GetModelStatusResponse* response,
     status_to_fill->mutable_status()->set_error_message(model_version_status.getErrorMsg());
 }
 
+void addStatusToResponse(tensorflow::serving::GetModelStatusResponse* response, const PipelineDefinitionStatus& pipeline_status) {
+    auto [state, error_code] = pipeline_status.convertToModelStatus();
+    SPDLOG_DEBUG("add_status_to_response state={} error_code", state, error_code);
+    auto status_to_fill = response->add_model_version_status();
+    status_to_fill->set_state(static_cast<tensorflow::serving::ModelVersionStatus_State>(static_cast<int>(state)));
+    status_to_fill->set_version(1);
```

Comment:
I don't like the hardcoded value here - it place logic of what pipeline definition version is in code that is for translating response.

What I propose is to pass the version to this function and in L100 have pipelineDefinitionVersion = pipelineDefinition->getVersion() wchich could point to static const model_version_t PipelineDefinition::VERSION;?

Reply:
Done